crates/sui-move-build/src/lib.rs

```diff
@@ -215,6 +220,18 @@ impl BuildConfig {
     }
 }
 
+fn report_linter_feedback_info(report: bool) -> anyhow::Result<()> {
+    if report {
+        if let Err(err) = writeln!(
+            &mut std::io::stderr(),
+            "Please report feedback on the linter warnings at https://forums.sui.io\n"
+        ) {
+            anyhow::bail!("Cannot report linter warnings feedback info: {}", err);
+        }
```

Comment:
When does writeln fail like this? I'd be fine with just doing 
```suggestion
        writeln!(
            &mut std::io::stderr(),
            "Please report feedback on the linter warnings at https://forums.sui.io\n"
        )?
```
But up to you :)

Reply:
I modeled it after how writing the diags buffer to stderr may fail (and how we report this error), but I don't mind making it simpler. Using eprintln! in this case (when not explicitly checking for error) would make even more sense probably.